### PR TITLE
Added new dir structure + df cleaning + logging 

### PR DESCRIPTION
**Pull request by AmritPatel**
Please review commit comments. Delete branch after PR merge and let's start anew.

@MattThomas9 still need:
- [ ] Test(s) for `laundry.py`
- [ ] Test(s) for `logger.py`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,11 +15,12 @@ A program to predict a home's sell price using data scraped from the internet.
...
-- `geopy`
-- `requests`
-- `pandas`
-- `configparser`
+- `beautifulsoup4` = "^4.9.3"
+- `geopy` = "^2.0.0"
+- `requests` = "^2.24.0"
+- `pandas` = "^1.1.3"
+- `configparser` = "^5.0.0"
+- `progress` = "^1.5"
```
Comment (MattThomas9) [inline README.md:23]: how you gonna add progress and not numpy???? @AmritPatel

File: hbf/rescrape/laundry.py [hunk 2]
```diff
@@ -51,6 +45,12 @@ def laundry(dirtyLaundry):
     dirtyLaundry["Year Built"] = dirtyLaundry["Year Built"].astype(float)
     dirtyLaundry["Lot Size"] = dirtyLaundry["Lot Size"].astype(float)
 
+    # Log clean SoldHomeData
+    logToFile(__name__, dirtyLaundry, 'INFO')
+    # Log clean SoldHomeData datatypes
+    logToFile(__name__, dirtyLaundry.dtypes, 'INFO')
+
     print(dirtyLaundry)
-    print(dirtyLaundry.dtypes)
```
Comment (MattThomas9) [inline hbf/rescrape/laundry.py]: this was literally just a debugging/test statement @AmritPatel .

File: README.md (hunk 1 above)
Comment (AmritPatel) [reply README.md:23]: `numpy` a sub-dependency

File: hbf/rescrape/laundry.py (hunk 2 above)
Comment (AmritPatel) [reply hbf/rescrape/laundry.py]: That is exactly what the log file is for!

**Pull request merged**